src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,658 @@
+import { expect } from 'chai';
+import { transform } from '../../utils/transform';
+import { sumValues, dateFormatter } from '../../utils/helpers';
+
+// Other functions to test
+// getFsrReason,
+// getMonthlyIncome,
+// getMonthlyExpenses,
+// getEmploymentHistory,
+// getTotalAssets,
+// filterDeductions,
+// otherDeductionsName,
+// otherDeductionsAmt,
+// nameStr,
+
+const someInputObject = {
```

Comment:
Organizational: I would put this in a separate file, its a ton of extra noise in the test file